apps/communications/contacts/js/views/list.js

```diff
@@ -1150,6 +1217,18 @@ contacts.List = (function() {
     } else {
       renderOrg(contact, link);
     }
+
+    // Additional Facebook data starts coming after we already
+    // rendered the list. At that point we already built the
+    // cache without this information. To avoid flashing the
+    // screen repainting it with this new information, we request
+    // a new cache refresh if we are adding the info to a node
+    // that belongs to the cache. Because we are throttling cache
+    // refresh requests, this shouldn't be too expensive.
+    // Life can be hard sometimes.
+    if (Cache.enabled && link.dataset.cache) {
```

Comment:
We do it only for the fb contacts that are on the cache, that means the ones above the fold (note the link.dataset.cache check). And in any case, cache requests are throttle :)